x/ccv/provider/keeper/distribution.go

```diff
@@ -127,6 +127,23 @@ func (k Keeper) AllocateTokens(ctx sdk.Context) {
 	}
 }
 
+// IsEligibleForConsumerRewards returns `true` if the validator with `consumerValidatorHeight` has been a consumer
+// validator for a long period of time and hence is eligible to receive rewards, and false otherwise
+func (k Keeper) IsEligibleForConsumerRewards(ctx sdk.Context, consumerValidatorHeight int64) bool {
+	numberOfBlocksToStartReceivingRewards := k.GetNumberOfEpochsToStartReceivingRewards(ctx) * k.GetBlocksPerEpoch(ctx)
+
+	// Note that at the beginning of the chain's life (e.g., when the block height of the chain is still small), no validator
+	// could have been a consumer validator for more than `NumberOfEpochsToStartReceivingRewards` epochs and hence in this
+	// case all validators are eligible for rewards, even though they might have not been a consumer validator on the
+	// consumer chain for more than a single epoch.
+	isChainStillNew := ctx.BlockHeight() < numberOfBlocksToStartReceivingRewards
```

Comment:
This check is for provider with height lower than 24*600. This is clearly an edge case that is not relevant for the Hub. I think we should remove it. Even in case the provider does start from height 0, the worst case scenario is that there are no rewards for ~24h.

Reply:
I removed the check.